.github/workflows/build_api.yml

```diff
@@ -0,0 +1,139 @@
+      - name: Rename artifacts
+        if: ${{ github.event.inputs.name != 'blank' }}
+        run: for f in $ARTIFACTS; do mv $f $(echo $f | sed -E 's/_0\.0\.0-SNAPSHOT-[^_]*/_${{ github.event.inputs.name }}/'); done
+      - name: List artifacts
+        run: ls -l api/dist
+      - name: Release nightly/rc
+        if: ${{ github.event.inputs.name != 'blank' }}
+        uses: ncipollo/release-action@v1
+        with:
+          allowUpdates: true
+          artifacts: ${{ env.ARTIFACTS }}
+          commit: ${{ github.event.inputs.sha }}
+          name: ${{ github.event.inputs.name }}
+          tag: ${{ github.event.inputs.name }}
+          body: ${{ github.event.inputs.sha_short }}
+          prerelease: true
+      - name: Create GitHub release
+        if: ${{ github.event.inputs.new_tag != 'blank' }}
+        uses: ncipollo/release-action@v1
+        with:
+          allowUpdates: true
+          artifacts: ${{ env.ARTIFACTS }}
+          commit: ${{ github.event.inputs.sha }}
+          name: ${{ github.event.inputs.new_tag }}
+          tag: ${{ github.event.inputs.new_tag}}
+          bodyFile: CHANGELOG_latest.md
+
+  build-docker-image:
+    name: Build and push Docker image
+    runs-on: ubuntu-latest
+    if: ${{ github.event.inputs.name != 'blank' || github.event.inputs.new_tag != 'blank' }}
+    env:
+      IMAGE_NAME: reearth/reearth-flow
+    defaults:
+      run:
+        working-directory: api
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.GH_APP_ID }}
+          private-key: ${{ secrets.GH_APP_PRIVATE_KEY }}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Get options
+        id: options
+        env:
+          TAG: ${{ github.event.inputs.new_tag_short && github.event.inputs.new_tag_short != 'blank' && github.event.inputs.new_tag_short || '' }}
+          NAME: ${{ github.event.inputs.name }}
+          SHA: ${{ github.event.inputs.sha_short }}
+        run: |
+          if [[ -n $TAG ]]; then
+            PLATFORMS=linux/amd64,linux/arm64
+            VERSION=$TAG
+            TAGS=$IMAGE_NAME:$TAG
+            if [[ ! $TAG =~ '-' ]]; then
+              TAGS+=,${IMAGE_NAME}:${TAG%.*}
+              TAGS+=,${IMAGE_NAME}:${TAG%%.*}
+              TAGS+=,${IMAGE_NAME}:latest
+            fi
+          else
+            PLATFORMS=linux/amd64
+            VERSION=$SHA
+            TAGS=$IMAGE_NAME:$NAME
+          fi
+          echo "::set-output name=platforms::$PLATFORMS"
+          echo "::set-output name=version::$VERSION"
+          echo "::set-output name=tags::$TAGS"
```

Comment:
The `set-output` command is deprecated, please migrate following this document 🙏 

Ref: [GitHub Actions: Deprecating save-state and set-output commands, GitHub](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)

Reply:
Resolved